docs/rules/bugs/deprecated-builtin.md

```diff
@@ -40,8 +40,88 @@ allow if input.user.is_root
 ## Rationale
 
 Calling deprecated built-in functions should always be avoided, and replacing them is usually trivial.
-Refer to the OPA docs on [strict mode](https://www.openpolicyagent.org/docs/latest/policy-language/#strict-mode)
-for more details on which built-in functions counts as deprecated.
+
+## Replacing Deprecated Built-in Functions
+
+### `any`
+
+Use the `in` keyword (OPA v0.34.0+) to replace the `any` function:
+
+**Instead of**
+```rego
+a := any([input.foo, input.bar])
+```
+
+**Do this**
+```rego
+import future.keywords.in # or `import rego.v1` (OPA v0.59.0+)
```

Comment:
is it worth a comment that this is natively supported in 1.0 without an import?

Reply:
The humongous notice at the top of the page should be enough, I think :)

<img width="914" alt="Screenshot 2025-01-27 at 10 47 03" src="https://github.com/user-attachments/assets/0ab89a9d-8cb3-40eb-9cdd-b2b908f4297c" />

Reply:
ahh wow, ok!